eth/config.go

```diff
@@ -47,7 +47,7 @@ var DefaultConfig = Config{
 	LightPeers:    100,
 	DatabaseCache: 768,
 	TrieCache:     256,
-	TrieTimeout:   5 * time.Minute,
+	TrieTimeout:   60 * time.Minute,
```

Comment:
In blockchain.go `NewBlockchain`, some other (hardcoded) defaults are used in case cacheconfig is `nil`. Perhaps they should use this value instead?

Reply:
Hmmm, I thought there would be used.